nautobot/core/testing/api.py

```diff
@@ -195,20 +195,26 @@ def test_options_object(self):
                 # Namings Help
                 # 1. detail_view_config: This is the detail view config set in the serializer.Meta.detail_view_config
                 # 2. detail_view_schema: This is the retrieve schema generated from an OPTIONS request.
+                # 3. advanced_view_schema: This is the advanced tab schema generated from an OPTIONS request.
                 serializer = get_serializer_for_model(self._get_queryset().model)
+                advanced_view_schema = response.data["view_options"]["advanced"]
+                self.assertEqual(len(advanced_view_schema), 1)
+                self.assertIn("Object Details", advanced_view_schema[0])
+
                 if detail_view_config := getattr(serializer.Meta, "detail_view_config", None):
                     detail_view_schema = response.data["view_options"]["retrieve"]
                     self.assertHttpStatus(response, status.HTTP_200_OK)
 
-                    # According to convention, `special_fields(`id`, `composite_key`, `url`)` should only exist at the end of the first col first group in
-                    # the `detail_view_schema` and should be deleted from any other places it may appear in the `detail_view_schema`. Assert this is True.
-                    with self.subTest("Assert special fields(`id`, `composite_key`, `url`)."):
-                        special_fields = ["id", "composite_key", "url"]
-
+                    # According to convention, fields in the advanced tab(`id`, `composite_key`, `url`, `created`, `last_updated`)` should not exist in
+                    # the `detail_view_schema`. Assert this is True.
+                    with self.subTest(
+                        "Assert advanced tab fields(`id`, `composite_key`, `url`, `created`, `last_updated`)."
+                    ):
+                        advanced_tab_fields = advanced_view_schema[0].get("Object Details")["fields"]
                         if detail_view_config.get("include_others"):
                             # Handle `Other Fields` specially as `Other Field` is dynamically added by nautobot and not part of the serializer view config
                             other_fields = detail_view_schema[0]["Other Fields"]["fields"]
-                            self.assertFalse(any(field for field in other_fields if field in special_fields))
+                            self.assertFalse(any(field for field in other_fields if field in advanced_tab_fields))
```

Comment:
Perhaps for readability:

```suggestion
                            for field in advanced_tab_fields:
                                self.assertNotIn(field, other_fields)
```

Similar suggestion below.